### PR TITLE
RSDK-2093 - default gpio motor PWM frequency to 800 Hz

### PR DESCRIPTION
**Pull request by gvaradarajan**

### CODE REVIEW

**Review by randhid — APPROVED** [review]
LGTM! Maybe check the default in one of the constructor tests and let's 
- [ ] try this out on a jetson once hardware pwm is in!

**Comment by randhid** [comment]
Did you want to change that comment about PWM frequency in `board/server.go`?

File: components/board/server.go [hunk 1]
```diff
@@ -138,8 +138,8 @@ func (s *subtypeServer) PWMFrequency(ctx context.Context, req *pb.PWMFrequencyRe
 	return &pb.PWMFrequencyResponse{FrequencyHz: uint64(freq)}, nil
 }
 
-// SetPWMFrequency sets a given pin of a board of the underlying robot to the given PWM frequency. 0 will use the board's default PWM
-// frequency.
+// SetPWMFrequency sets a given pin of a board of the underlying robot to the given PWM frequency.
+// For Raspberry Pis, 0 will use a default PWM frequency of 800.
```
Comment (penguinland) [inline components/board/server.go:142]: :+1: This is an excellent way to rephrase! Thanks for making this change.

**Comment by gvaradarajan** [comment]
Tested on RPi and Jetson Orin, default is set appropriately

**Pull request merged**